tests/unit/test_locationcodes.py

```diff
@@ -0,0 +1,82 @@
+"""Tests for strtools.py"""
+
+import time
+import random
+import importlib
+
+import pytest
+from scipy import stats
+from zolltools.nemsis import locationcodes
+
+
+@pytest.mark.slow
+def test_get_mapping_performance():
+    """
+    measuring performance of get_mapping
+
+    Takes a sample of the execution time for the first and successive calls to
+    `get_mapping` and determines that successive calls are at least
+    `min_exp_speedup` times faster than the first calls within a certain
+    confidence interval (see `alpha` and comparison to p-value in `assert`
+    statement).
+
+    `min_exp_speedup` was determined with preliminary testing. See gh 74
+    """
+
+    alpha = 0.05 # confidence threshold
+    min_exp_speedup = 10 # successive reads should be min_exp_speedup times faster
```

Comment:
```suggestion
    min_exp_speedup = 10
```

Already explained by docstring

Reply:
resolved by new changes